playbooks/tasks/lastpass-save-cluster-config.yml

```diff
@@ -49,10 +49,44 @@
           and your config files and then rerun this playbook.
       when: update_lastpass_result.user_input[:1] not in 'yY'
 
+    - name: Keep better yaml formatting for mongo_db_mgkey
+      block:
+        - name: Store mongo mgkey to temporary variable and indent it
+          vars:
+            # Prefix that will be rendered in final LastPass yaml
+            prefix: "  "
+          set_fact:
+            # Add prefix to each mgkey line
+            indented_mgkey: >-
+              {{ [prefix]
+              | product(webportal_common_config_last.mongo_db_mgkey.splitlines())
+              | map('join')
+              | list
+              | join('\n') }}
+
+        - name: Update last config with a dummy mongo mgkey value
+          set_fact:
+            webportal_common_config_last: >-
+              {{ webportal_common_config_last
+              | combine({'mongo_db_mgkey': 'dummy'}) }}
+
+        - name: Create last config yaml string
+          set_fact:
+            common_config_last_yaml: >-
+              {{
+                webportal_common_config_last
+                | to_nice_yaml(width=2048)
+                | replace(
```

Comment:
REQ: code like this needs comments. Is using replace like this some sort of work around? It doesn't make sense why we would write a dummy value only to overwrite it immediately after.

Reply:
See above.